lib/MHA/Config.pm

```diff
@@ -263,6 +263,12 @@ sub parse_server {
   $value{skip_init_ssh_check} = 0
     if ( !defined( $value{skip_init_ssh_check} ) );
 
+  $value{skip_version_check} = $param_arg->{skip_version_check};
+  $value{skip_version_check} = $default->{skip_version_check}
+    if ( !defined( $value{skip_version_check} ) );
```

Comment:
It's the same pattern as is used for SSH port configuration, for example: https://github.com/insom/mha4mysql-manager/blob/30f28857c09a7a8399ec054231c468e211327571/lib/MHA/Config.pm#L86:L90